main.nf

```diff
@@ -203,6 +222,32 @@ process get_software_versions {
     """
 }
 
+/*
+ * Generate BAM files if input files are CRAM files
+ */
+
+if ( params.cram_files ) {
+
+  process cramToBamWithReference {
```

Comment:
I think it should be possible to use samtolls fastq directly with cram files. http://www.htslib.org/doc/samtools-fasta.html Possibly not easy in DSL1, in that case could leave it as is and update once I get to convert this to DSL2